File: C#/Remove Duplicates from Sorted List.cs
/**
 * Definition for singly-linked list.
 * public class ListNode {
 *     public int val;
 *     public ListNode next;
 *     public ListNode(int val=0, ListNode next=null) {
 *         this.val = val;
 *         this.next = next;
 *     }
 * }
 */
public class Solution83 {
    public ListNode DeleteDuplicates(ListNode head) {
        
        if(head == null || head.next == null)
            return head;
        
        ListNode list = head;
        
        while(list.next!=null)
        {
            if(list.val==list.next.val)
            {
                list.next = list.next.next;
            }
            else
            {
                list = list.next;
            }
        }
        
        return head;
    }
}

File: Golang/Remove Duplicates from Sorted List.go
//1
/**
 * Definition for singly-linked list.
 * type ListNode struct {
 *     Val int
 *     Next *ListNode
 * }
 */
func deleteDuplicates(head *ListNode) *ListNode {
    curr := head

    if head == nil || head.Next == nil {
        return head
    }

    for curr.Next != nil {
        if curr.Val == curr.Next.Val {
            curr.Next = curr.Next.Next
        } else {
            curr = curr.Next
        }
    }

    return head
}

//2
/**
 * Definition for singly-linked list.
 * type ListNode struct {
 *     Val int
 *     Next *ListNode
 * }
 */
func deleteDuplicates(head *ListNode) *ListNode {
    cur := head
    for cur != nil {
        if cur.Next != nil && cur.Val == cur.Next.Val {
            cur.Next = cur.Next.Next
            continue
        }
        cur = cur.Next
    }
    
    return head
}